webknossos-datastore/conf/com.scalableminds.webknossos.datastore.routes

```diff
@@ -16,18 +16,26 @@ GET           /datasets/:organizationName/:dataSetName/layers/:dataLayerName/his
+GET           /zarr/:organizationName/:dataSetName/:dataLayerName/:mag                                                              @com.scalableminds.webknossos.datastore.controllers.ZarrStreamingController.requestDataLayerMagFolderContents(token: Option[String], organizationName: String, dataSetName: String, dataLayerName: String, mag: String)
+GET           /zarr/:organizationName/:dataSetName/:dataLayerName/:mag/.zarray                                                      @com.scalableminds.webknossos.datastore.controllers.ZarrStreamingController.requestZArray(token: Option[String], organizationName: String, dataSetName: String, dataLayerName: String, mag: String)
+GET           /zarr/:organizationName/:dataSetName/:dataLayerName/:mag/:cxyz                                                        @com.scalableminds.webknossos.datastore.controllers.ZarrStreamingController.requestRawZarrCube(token: Option[String], organizationName: String, dataSetName: String, dataLayerName: String, mag: String, cxyz: String)
+
+GET           /zarr_private/:accessToken                                                                                  @com.scalableminds.webknossos.datastore.controllers.ZarrStreamingController.dataSourceFolderContentsPrivateLink(accessToken: String)
+GET           /zarr_private/:accessToken/.zgroup                                                                          @com.scalableminds.webknossos.datastore.controllers.ZarrStreamingController.zGroupPrivateLink(accessToken: String,  dataLayerName="")
+GET           /zarr_private/:accessToken/datasource-properties.json                                                       @com.scalableminds.webknossos.datastore.controllers.ZarrStreamingController.dataSourceWithAnnotationPrivateLink(accessToken: String)
+GET           /zarr_private/:accessToken/:dataLayerName                                                                   @com.scalableminds.webknossos.datastore.controllers.ZarrStreamingController.dataLayerFolderContentsPrivateLink(accessToken: String,  dataLayerName: String)
+GET           /zarr_private/:accessToken/:dataLayerName/.zattrs                                                           @com.scalableminds.webknossos.datastore.controllers.ZarrStreamingController.zAttrsWithAnnotationPrivateLink(accessToken: String, dataLayerName: String)
+GET           /zarr_private/:accessToken/:dataLayerName/.zgroup                                                           @com.scalableminds.webknossos.datastore.controllers.ZarrStreamingController.zGroupPrivateLink(accessToken: String,  dataLayerName: String)
+GET           /zarr_private/:accessToken/:dataLayerName/:mag                                                              @com.scalableminds.webknossos.datastore.controllers.ZarrStreamingController.dataLayerMagFolderContentsPrivateLink(accessToken: String,  dataLayerName: String, mag: String)
```

Comment:
please resolve the merge conflicts with the already-in-master pr https://github.com/scalableminds/webknossos/pull/6366 – the new thing there is that the directory listing routes should exist both with and without a trailing slash. This should be the case for your new routes as well